src/components/views/post-list.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import './post-list.scss';
+
+class PostList extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        const { posts } = this.props;
+        return (
+            <div>{posts.map(post => {
+                let date = new Date(post.date * 1000);
+
+                return (
+                    <article key={post.id} className="article-body">
+                        <header>
+                            <div className="article-date">{date.toDateString()}</div>
+                            <h1 className="article-title">{post.title}</h1>
+                            <div className="article-info">
+                                <a href="#">{post.author}</a> ● {post.tags.map(tag => {return <a href="#" key={tag.toString()}><span>{tag}</span></a>})}
```

Comment:
I think that 'dot' symbole could be applied with styles ?
Make this line of code more readable moving its parts to the new lines